presidio-anonymizer/presidio_anonymizer/engine/anonymizers/fpe.py

```diff
@@ -0,0 +1,17 @@
+# TODO implement + test
+from anonymizers.anonymizer import AnonymizerAbstract
+
+
+class FPE(AnonymizerAbstract):
```

Comment:
FPE stands for Format preserving encryption? 
Do we want to implicitly state that?

Reply:
Yes. 
I don't mind changing it but I think it is too long and FPE is a standard WDYT?

Reply:
I think descriptiveness is worth the long name, I wan't aware of this terminology

Reply:
I will add a remark.

Reply:
FPE is good enough. Please add docstring descring the class - there the full trasformation name can be used